.github/ISSUE_TEMPLATE/----bug-report---.md

```diff
@@ -35,4 +35,4 @@ If applicable, add screenshots to help explain your problem.
  - Version [e.g. 22]
```

Comment:
@Programming-School-Pro-Coding Same problem in this PR. Please do not commit the unnecessary changes.

Also create an issue for this play. Thanks

Reply:
What is the lines do you want to remove ?

Reply:
Why do you need to change the files inside the .github folder. The code change diff shows those file have changed.

It is happing because the branch you started working is not clean.

Reply:
@Programming-School-Pro-Coding Please correct the above mistakes before I start review